timesketch/lib/datastores/elastic.py

```diff
@@ -282,6 +285,51 @@ def search(self,
             _source_include=return_fields,
             scroll=scroll_timeout)
 
+    def search_stream(
+            self, sketch_id=None, query_string=None, query_filter=None,
+            query_dsl=None, indices=None, return_fields=None):
+        """Search ElasticSearch. This will take a query string from the UI
+        together with a filter definition. Based on this it will execute the
+        search request on ElasticSearch and get result back.
+
+        Args :
+            sketch_id: Integer of sketch primary key
+            query_string: Query string
+            query_filter: Dictionary containing filters to apply
+            query_dsl: Dictionary containing Elasticsearch DSL query
+            indices: List of indices to query
+            return_fields: List of fields to return
+
+        Returns:
+            Generator of event documents in JSON format
+        """
+
+        if not query_filter.get(u'limit'):
+            query_filter[u'limit'] = 10000
```

Comment:
Move this to a class constant or similar

Reply:
Done.